lib/game.rb

```diff
@@ -0,0 +1,7 @@
+class Game
+
+  def random_option
+    ["Rock", "Paper", "Scissors"].sample
+  end
+
+end
```

Comment:
could have kept more logic of game in this class :) i.e. deciding who wins and how to make players keep their weapons.